### DISCUSSION TITLE
SR users able to interact with elements behind annotations modal after making an edit/reply

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3481] [Accessibility]
Priority level 1 (out of 3) from UW Accessible Technologies

WCAG mapping: 2.1.2 No Keyboard Trap

STEPS TO REPRODUCE:
1. Select a book (i.e. "I My Recall")
2. Turn on caret browsing (insert + F7)
3. Turn on a screen reader.
4. Tab into the text to an annotation and click enter.
5. Navigate to a comment in this annotation modal, make an edit, and save the changes.
6. Use down arrow and note how the screen reader is reading content from behind the annotation modal.

OBSERVED BEHAVIOR
After doing an action in the annotation modal such as making an edit or reply, focus is reset to the top of the page and screen reader users are able to interact with the content behind the modal. 

The screen shot was recorded using JAWS/Chrome

EXPECTED BEHAVIOR:
Screen reader user's focus should always be trapped inside a modal if the modal is open. If it is not trapped, then screen reader users may not be aware that the modal is open.

**Comment by dananjohnson** [comment]
I believe this issue is resolved. We made a number of improvements to focus management around annotations for [v8](https://manifoldscholar.github.io/manifold-docusaurus/docs/administering/release_notes/v8x).
